pkgs/by-name/ws/wstunnel-rust/package.nix

```diff
@@ -0,0 +1,32 @@
+{ fetchFromGitHub
+, rustPlatform
+, lib
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wstunnel-rust";
+  version = "9.6.1";
+
+  src = fetchFromGitHub {
+    owner = "erebe";
+    repo  = "wstunnel";
+    rev = "v${version}";
+    sha256 = "sha256-+tNQf53KUNIz9RFVTHqgXEKLxlXiQKKPZtXqD3C/t7I=";
+  };
+
+  cargoHash = "sha256-NER7NIWUXyWFUGcdZ5HLFaKaO9FNLPlG7jql08karS4=";
+
+  checkFlags = [
+    # make use of network connection
+    "--skip=tcp::tests::test_proxy_connection"
+  ];
+
+  meta = with lib; {
+    description = "Wstunnel uses the websocket protocol which is compatible with http in order to bypass firewalls and proxies.";
```

Comment:
Description should not contain package name, should be short and concise, and shouldn't end with a period.